sympy/polys/series/base.py

```diff
@@ -109,6 +109,18 @@ def equal_repr(self, s1: TSeries, s2: TSeries, /) -> bool:
         """Check if two power series have the same representation."""
         ...
 
+    def is_element(self, arg: TSeries) -> bool:
+        """Check if a arg is an element of the power series ring."""
+        ...
+
+    def is_ground(self, arg: TSeries | Er) -> bool | None:
+        """Check if a arg is a ground element of the power series ring."""
+        ...
+
+    def leading_coefficient(self, s: TSeries) -> Er:
+        """Return the leading coefficient of a power series."""
```

Comment:
I think it is better to call this the constant coefficient rather than the leading coefficient. For polynomials LC is the coefficient for the term of highest degree.